utils/cliutils/utils.go

```diff
@@ -516,30 +516,37 @@ func CreateConfigCmd(c *cli.Context, confType artifactoryUtils.ProjectType) erro
 
 func RunNativeCmdWithDeprecationWarning(cmdName string, projectType artifactoryUtils.ProjectType, c *cli.Context, cmd func(c *cli.Context) error) error {
```

Comment:
For consistency ad readability reasons, I suggest we make this function more similar to the ```RunConfigCmdWithDeprecationWarning``` function, in the sense that the log warning message is placed in a different function.